packages/message-kit/src/helpers/types.ts

```
@@ -1,13 +1,13 @@
-import { default as HandlerContext } from "../lib/handlerContext.js";
+import { HandlerContext } from "../lib/handlerContext.js";
 import { ClientOptions } from "@xmtp/node-sdk";
 import { ContentTypeId } from "@xmtp/content-type-primitives";
 
 export type MessageAbstracted = {
   id: string;
   sent: Date;
-  content: any;
+  content: { text: string } | {};
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)